File: PSET6/mario/more/mario.py
from cs50 import get_int

# ask for height between 1 and 8
while True:
    height = get_int("Height: ")
    if height in range(1, 9):
        y = 0
        for y in range(height):
            length = 0
            spaces = 0
            x = 0
            for spaces in range(height - y - 1):
                print(" ", end="")
                spaces += 1
            for length in range(y + 1):
                print("#", end="")
                length += 1
            print("  ", end="")
            for x in range(y + 1):
                print("#", end="")
            print()
            y += 1
        exit(0)

File: PSET1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h, r, c, space;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    for (r = 0; r < h; r++)
    {
        for (space = 0; space < h - r - 1; space++)
        {
            printf(" ");
        }
        for (c = 0; c <= r; c++)
        {
            printf("#");
        }
        printf("\n");
    }
}